Sources/TuistAutomation/Utilities/TargetBuilder.swift

```diff
@@ -131,4 +150,41 @@ public final class TargetBuilder: TargetBuilding {
                 try FileHandler.shared.copy(from: product, to: productOutputPath)
             }
     }
+
+    private func findDestination(
```

Comment:
This code was copy/pasted from `TestService` - does it make sense to extract it into its own object? Something like `XcodeBuildDestinationFinder`? Not sure if it's worth the effort given that it's only the second place we need this code, but figured it was worth calling out.

Reply:
What about making this an extension method of `XcodeBuildDestination`, so it can be shared between the two?

Reply:
@danyf90 done!